runtime/gc_realtime/RealtimeGC.cpp

```diff
@@ -1036,13 +484,13 @@ MM_RealtimeGC::reportGCCycleFinalIncrementEnding(MM_EnvironmentBase *env)
  * @ingroup GC_Metronome methodGroup
  */
 void
-MM_RealtimeGC::reportSyncGCStart(MM_EnvironmentBase *env, GCReason reason, UDATA reasonParameter) 
+MM_RealtimeGC::reportSyncGCStart(MM_EnvironmentBase *env, GCReason reason, uintptr_t reasonParameter)
 {
-	PORT_ACCESS_FROM_ENVIRONMENT(env);
-	UDATA approximateFreeFreeMemorySize;
-#if defined(J9VM_GC_DYNAMIC_CLASS_UNLOADING)
+	OMRPORT_ACCESS_FROM_ENVIRONMENT(env);
+	uintptr_t approximateFreeFreeMemorySize;
+#if defined(OMR_GC_DYNAMIC_CLASS_UNLOADING)
 	MM_ClassUnloadStats *classUnloadStats = &_extensions->globalGCStats.classUnloadStats;
-#endif /* defined(J9VM_GC_DYNAMIC_CLASS_UNLOADING) */
```

Comment:
This is inconsistent with other code now. At the end of refactoring MM_RealtimeGC code should not have OMR_GC_DYNAMIC_CLASS_UNLOADING lines. However you can do this cleanup later.